instrumentation/faraday/test/opentelemetry/instrumentation/faraday/middlewares/tracer_middleware_test.rb

```diff
@@ -41,21 +42,32 @@
   describe 'first span' do
     before do
       instrumentation.install
+      instrumentation.instance_variable_set(:@installed, false)
+      instrumentation.install(hide_query_params: true)
     end
 
     it 'has http 200 attributes' do
-      response = client.get('/success')
+      response = client.get('/success?foo=bar')
 
       _(span.name).must_equal 'HTTP GET'
       _(span.attributes['http.method']).must_equal 'GET'
       _(span.attributes['http.status_code']).must_equal 200
-      _(span.attributes['http.url']).must_equal 'http://example.com/success'
+      _(span.attributes['http.url']).must_equal 'http://example.com/success?'
```

Comment:
IDK whether this was hashed out on a previous PR... I feel like we should drop the `?` here as well. Alternatively, append something to make it clear the query parameters have been redacted - e.g. `?<redacted>`.